### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by brucearctor**
# TL;DR
Update org references

## Type
 - [x] Bug Fix

## Are all requirements met?

 - [x] Code completed
 
## Complete description
update org references

## Tracking Issue
We didn't make one for this...  I should create one for all.

### CODE REVIEW

**Comment by brucearctor** [comment]
@EngHabu -- interesting... passed the tests, but not lint (also, I'm used to seeing linters before the unit tests, not after).  

It looks like you had something special with mockery and your own repo.  I'll keep thinking of approaches (and I'm probably making a mess of other things, lol ;-) )

File: boilerplate/lyft/golang_test_targets/download_tooling.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ set -e
 # List of tools to go get
 # In the format of "<cli>:<package>" or ":<package>" if no cli
 tools=(
-  "github.com/vektra/mockery/cmd/mockery"
-  "github.com/lyft/flytestdlib/cli/pflags"
+  "github.com/vektra/mockery/cmd"
```
Comment (EngHabu) [inline boilerplate/lyft/golang_test_targets/download_tooling.sh:19]:
```suggestion
  "github.com/vektra/mockery/cmd/mockery"
```

File: boilerplate/lyft/golang_test_targets/download_tooling.sh [hunk 2]
```diff
@@ -16,8 +16,8 @@ set -e
...
+  "github.com/flyteorg/flytestdlib/cli/pflags"
```
Comment (EngHabu) [inline boilerplate/lyft/golang_test_targets/download_tooling.sh:20]: So the go.mod and go.sum files in this folder refer to an "older" version of pflags... a 3.11 version which is still called github.com/lyft/* 

I think the easiest would be to omit changes to the golang_test_targets folder completely... let the rest of your PR get merged and a new release is created, then go back and update just that folder...

Comment (brucearctor) [reply boilerplate/lyft/golang_test_targets/download_tooling.sh:20]: Agreed.  I was trying to figure out how to do this all in one PR and that seemed a bit overkill to force that issue.

**Comment by brucearctor** [comment]
@EngHabu -- also, I'm wondering why boilerplate is in the repo.  Shouldn't we make this its own repo/module (distinct from flytestdlib, flytectl, flyteidl, etc).  It looks like this is ~approx the same in each repo.

**Comment by brucearctor** [comment]
@EngHabu  -- OMG, it is GREEN.

**Review by EngHabu — APPROVED** [review]

**Pull request merged**